### DISCUSSION TITLE
A Java Runtime Environment (JRE) or Java Development Kit (JDK) must be available in order to run Dbeaver.

### DISCUSSION THREAD

**Discussion by ori-bibas** [dbeaver/dbeaver#22751] [DBeaver Development]
Hello, 

After running the build script and running the binaries, I am receiving this error on start-up:

<img width="280" alt="image" src="https://github.com/dbeaver/dbeaver/assets/68831703/19da23ea-4aa6-4c0f-9c3d-24c6d3934163">

Any help?

**Comment by mkgrgis** [comment]
@ori-bibas , what about short environment description?
